### PR TITLE
Confluent Cloud dashboard update

### PR DESCRIPTION
**Pull request by 4demos**
# Summary

update to Confluent Cloud dashboard

### Dashboards

- [x] Does the PR contain a screenshot for each of your dashboards?
- [x] Do your screenshots show data?
- [x] Has the [sanitization script](https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#dashboards) been run on each dashboard?

### CODE REVIEW

**Review by caylahamann — APPROVED** [review]

**Comment by 4demos** [comment]
"ERROR: The following errors occurred while validating: quickstarts/confluent-cloud/config.yml
- quickstartMetadata/dashboards/rawConfiguration/variables/nrqlQuery/accountIds: `accountIds` can't be blank" 
accountIds not used in confluent-cloud.json, they all replaced with accountId and set to 0

File: dashboards/confluent-cloud/confluent-cloud.json [hunk 1]
```diff
@@ -1,622 +1,633 @@
...
+      ]
+    }
+  ],
+  "variables": [
+    {
+      "name": "kafka_id",
+      "items": null,
+      "defaultValues": [],
+      "nrqlQuery": {
+        "accountId": 0,
```
Comment (aswanson-nr) [inline dashboards/confluent-cloud/confluent-cloud.json:624]: Hey @4demos, it looks like this is the culprit. If we convert it over to `accountIds` version it should work just fine.
```suggestion
        "accountIds": [],
```

**Review by aswanson-nr — CHANGES_REQUESTED** [review]
Hey @4demos, Figured out where that workflow was failing!

File: dashboards/confluent-cloud/confluent-cloud.json (hunk 1 above)
Comment (4demos) [reply dashboards/confluent-cloud/confluent-cloud.json]: good catch! 
still not sure why in most cases we replace `"accountIds": [...]` with `"accountId": 0` but here we should keep "accountIds"

**Review by brammerl — APPROVED** [review]

File: dashboards/confluent-cloud/confluent-cloud.json (hunk 1 above)
Comment (aswanson-nr) [reply dashboards/confluent-cloud/confluent-cloud.json]: Agreed! Did the dashboard cleanup script adjust it to `"accountId": 0`?

**Review by aswanson-nr — APPROVED** [review]

File: dashboards/confluent-cloud/confluent-cloud.json (hunk 1 above)
Comment (4demos) [reply dashboards/confluent-cloud/confluent-cloud.json]: `sanitize-dashboard` doesn't set "accountId": 0, just removes the actual account. But instructions https://developer.newrelic.com/contribute-to-quickstarts/build-a-quickstart/add-dashboard/ tell submitters: "Set your accountId to 0". If it's not a requirement any longer and `"accountIds": []` is fine, may be it's time to update the docs?

Comment (aswanson-nr) [reply dashboards/confluent-cloud/confluent-cloud.json]: @4demos, thank you! I'll get the docs updated

**Pull request merged**